lib/video/screenRecording/firefox/firefoxWindowRecorder.js

```diff
@@ -174,6 +175,9 @@ module.exports = class FirefoxWindowRecorder {
     log.info('Stop firefox window recorder.');
     await enableWindowRecorder(false, this.browser);
 
+    // Add some extra delay to make sure the recording is finished.
+    await delay(this.options.firefox.windowRecorderStopDelay || 1000);
```

Comment:
There is also a delay for Android video recordings: https://github.com/sitespeedio/browsertime/blob/9004781fda367edcbd7285723949a02c23ec998f/lib/video/screenRecording/android/recorder.js#L14

Would it be possible to generalize these delays?

Reply:
Sounds good, but lets wait first if there's way to fix the window recorder. We could probably rework both timoutes and wait times to just make it simpler.